ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -235,3 +235,29 @@ def test_numpy_multinomial(
         pvals=np.array([1 / n] * n, dtype=dtype[0]),
         size=size,
     )
+
+
+# permutation
+@handle_frontend_test(
+    fn_tree="numpy.random.permutation",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
probably need to add `min_num_dims=1` here so that arrays with zero dimensions are not generated :slightly_smiling_face: